authn/saml/src/main/java/se/swedenconnect/signservice/authn/saml/config/MetadataProviderConfiguration.java

```diff
@@ -58,6 +59,9 @@ public class MetadataProviderConfiguration {
   /**
    * Optional property. If {@code url} is assigned, this setting tells where a backup of the downloaded data should be
    * saved.
+   * <p>
+   * If the {@code mdq} flag has been set, this property should point to a directory and not a file.
+   * </p>
```

Comment:
This is a nit, but may be a bit confusing. The "backupFile" is in this case a backup directory as this metadata source is no longer represented by a single file.
Whouldn't it be better to rename this to something like "backupResource" or "backupLocation"

Reply:
Yes, I agree